modules/vorbis/audio_stream_ogg_vorbis.cpp

```diff
@@ -460,6 +460,23 @@ void AudioStreamOggVorbis::maybe_update_info() {
 		ERR_FAIL_COND_MSG(err != 0, "Error parsing header packet " + itos(i) + ": " + itos(err));
 	}
 
+	Dictionary dictionary;
+	for (int i = 0; i < comment.comments; i++) {
+		String c = String::utf8(comment.user_comments[i]);
+		int equals = c.find_char('=');
+
+		if (equals == -1) {
+			WARN_PRINT("Invalid vorbis comment.");
```

Comment:
```suggestion
			WARN_PRINT("Invalid comment in Ogg Vorbis file.");
```

I don't know if you can get the file path from this method, but if you can, it would be good to display the path to the file as well.